src/Clipboard.ts

```diff
@@ -53,6 +53,27 @@ export const Clipboard = {
   setString(content: string) {
     NativeClipboard.setString(content);
   },
+
+  setImage(imageUrl: string) {
+    fetch(imageUrl)
+      .then((response) => {
+          if (!response.ok) { throw Error(response.statusText); }
+          return response.blob();
+      })
+      .then( blob => new Promise( callback =>{
```

Comment:
```suggestion
      .then(blob: Blob => new Promise(() => {
```

`callback` seems unused.